php/Terminus/Helpers/Input.php

```diff
@@ -302,31 +303,32 @@ public static function orgId(array $arg_options = array()) {
 
     $arguments = $options['args'];
     $key       = $options['key'];
-    if (!isset($arguments[$key]) && isset($_SERVER['TERMINUS_ORG'])) {
-      $arguments[$key] = $_SERVER['TERMINUS_ORG'];
-    }
-
-    $org_list = self::orgList($options);
-    $flip    = array_flip($org_list);
+    $org_list  = self::orgList($options);
     if (isset($arguments[$key])) {
-      if (isset($flip[$arguments[$key]])) {
-        return $flip[$arguments[$key]];
-      } elseif (isset($org_list[$arguments[$key]])) {
-        return $arguments[$key];
-      } elseif (in_array($arguments[$key], self::$NULL_INPUTS)
-        || !empty($arguments)
-      ) {
+      if ($id = array_search($arguments[$key], $org_list)) {
+        return $id;
+      }
+      return $arguments[$key];
+    } else if (isset($_SERVER['TERMINUS_ORG'])) {
+      return $_SERVER['TERMINUS_ORG'];
```

Comment:
This one is imported from `.env` or used with `TERMINUS_ORG=<org name or id> terminus foo bar`. It def won't be in the constants file unless someone added it themselves. Perhaps I misunderstand.